### PR TITLE
[CIS-343] Improve v3 SwiftUI sample's ChatView

### PR DESCRIPTION
**Pull request by cardoso**
~Base is #485 for now.~ moved to master

Added most features present in the UIKit sample.

Additionally:
- Added random user button to the login screen - this helps testing :)

### CODE REVIEW

File: Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift [hunk 1]
```diff
@@ -7,38 +7,119 @@ import SwiftUI
 
 @available(iOS 13, *)
 struct ChatView: View {
-    @State var channelId: ChannelId
+    /// SwiftUI expects view initialization to be cheap, so we hold the a `ChannelController` builder to be called in `onAppear`.
+    let channelBuilder: () -> ChannelController.ObservableObject
```
Comment (VojtaStavik) [inline Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift:11]: Let's wait for #488 to be merged. After that, you can remove this and just make `synchronize()` call in `onAppear`.

Comment (VojtaStavik) [reply Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift:11]: cc @mantergo

**Comment by cardoso** [comment]
The synchronize change works great!

File: Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift [hunk 2]
```diff
@@ -7,38 +7,111 @@ import SwiftUI
...
+                .default(Text("Add Member"), action: { self.channel.controller.addMembers(userIds: userIds) }),
+                .default(Text("Remove Member"), action: { self.channel.controller.removeMembers(userIds: userIds) }),
+                .cancel()
+            ])
+        }
     }
 }
+
+/// Need to conform to Indentifiable for action sheet
+extension Message: Identifiable {}
```
Comment (VojtaStavik) [inline Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift:117]: Can you please move this to our SDK? The same way we do it for `ChannelModel`

**Review by VojtaStavik — APPROVED** [review]
LGTM 🚢 🇮🇹

File: Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift (hunk 2 above)
Comment (cardoso) [reply Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift:117]: @VojtaStavik I see that conformance is not needed anymore after I made a change to the action sheet logic (to support action sheet for messages and the channel at the same time, the trigger was changed from Message to a Bool). So I'm removing for now.

**Pull request merged**